packages/statusbar-extension/src/index.ts

```diff
@@ -355,6 +357,26 @@ const modeSwitch: JupyterFrontEndPlugin<void> = {
     shell.modeChanged.connect((_, mode) => {
       modeSwitch.value = mode === 'single-document';
     });
+
+    if (settingRegistry) {
+      const loadSettings = settingRegistry.load(STATUSBAR_PLUGIN_ID);
+      const updateSettings = (settings: ISettingRegistry.ISettings): void => {
+        const startWithSimpleMode = settings.get('startWithSimpleMode')
+          .composite as boolean;
+        shell.mode = startWithSimpleMode
+          ? 'single-document'
+          : 'multiple-document';
```

Comment:
I can try again with that option, but this was generating some issues (difficult to explain here) related to toogle state not reflecting the option.

However, I was thinking that instead of a boolean, we could go with `startingMode` which could be blank (then the url or whatever latest state comes in the picture), `single` or `multiple` in which case the current code would act.

If you like this approach, I can give it a try.

Reply:
My last implements implements that approach which works fine for me. Use can force the startup mode via setting, or if that setting is blank, the URL enforce the startup mode.